composer.json

```diff
@@ -39,17 +39,17 @@
         "symfony/polyfill-uuid": "^1.13.1"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "^2.17",
+        "friendsofphp/php-cs-fixer": "^2.19|^3.4",
```

Comment:
This is not ideal, since the config will break under 2.x, but that's not a big problem..

Reply:
Yes I know, but `3.x` does not support Symfony `3.4` anymore and it didn't make sense to do more work to have two distinct config files as the older version is unmaintained and we always run the tool with the newest dependencies